Add missing includes required for Qt 5 build to succeed.

Author: David Carne <davidcarne@gmail.com>
Gentoo-bug: 587948

--- a/src/libkst/editablematrix.cpp
+++ b/src/libkst/editablematrix.cpp
@@ -19,6 +19,7 @@
 #include "debug.h"
 #include <qbytearray.h>
 #include <QXmlStreamWriter>
+#include <QDataStream>
 
 namespace Kst {
 
--- a/src/libkst/editablevector.cpp
+++ b/src/libkst/editablevector.cpp
@@ -18,6 +18,7 @@
 // Create QDataStream into a QByteArray
 // qCompress the bytearray
 #include <QXmlStreamWriter>
+#include <QDataStream>
 
 #include "debug.h"
 namespace Kst {
